cypress/integration/learn.js

```diff
@@ -1,8 +1,8 @@
 const CANONICAL_URL = 'https://www.mongodb.com/developer/learn/';
 const FIRST_ARTICLE_IN_ORDERING = '/how-to/transactions-c-dotnet/';
-const FIRST_ARTICLE_UPDATED_DATE = 'Oct 03, 2020';
+const FIRST_ARTICLE_UPDATED_DATE = 'Nov 18, 2025';
```

Comment:
the date seem incorrect to me. It should reflect the strapi article's first updated date isn't it?

Reply:
Also one clarification - STRAPI_PUBLICATION_STATE=live is set to true by default correct, so we are only pulling cms-test articles that are published for running these tests, correct?